src/components/Table/HeadingCell.react.js

```diff
@@ -3,11 +3,47 @@ export class HeadingCell extends React.Component {
     super(props);
   }
 
+  queryName() {
+    return "order[" + this.props.sortable + "]";
+  }
+
+  currentSortOrder() {
+    return this.context.tableStore.getUrl().query(true)[this.queryName()];
+  }
+
+  nextSortOrder() {
+    return this.currentSortOrder() === "asc" ? "desc" : "asc";
```

Comment:
i'm assuming this is for the reversing of the sort order... could we rename the method to something like reversedSortOrder?